### DISCUSSION TITLE
Metrics shown on highlight PRs

### DISCUSSION THREAD

**Discussion by CBID2** [open-sauced/feedback#8] [Feature request]
Since OpenSauced now has the option to highlight pull requests, it would be nice to have a feature where the positive impact is displayed (e.g., amount repo of visitors). This would be helpful for users to describe their experiences in interviews.

**Comment by BekahHW** [comment]
@CBID2 can you talk a little bit more about what you mean? I'm not sure I quite understand what we'd need to implement that feature.

**Reply by CBID2** [reply]
I mean like have some sort way that the PR could show how it helped a project. It's difficult to explain on a technical/code front @BekahHW.

**Reply by BekahHW** [reply]
Can you give an example of what you mean?

**Reply by CBID2** [reply]
> Can you give an example of what you mean?

Something like this: ![idea](https://github.com/open-sauced/feedback/assets/105683440/9679a5c8-ea88-4f94-b8cd-62fa699a2ae1).

Does it make more sense now @BekahHW?

**Reply by BekahHW** [reply]
I think one of the hard parts about this is that some PRs will have no immediate impact and the type of impact will vary on a case-by-case basis.

**Reply by CBID2** [reply]
Ooh good point
